sdcm/sct_config.py

```diff
@@ -1221,7 +1221,9 @@ class SCTConfiguration(dict):
         dict(name="data_volume_disk_iops", env="SCT_DATA_VOLUME_DISK_IOPS",
              type=int,
              help="Number of iops for ebs type io2|io3|gp3"),
-
+        dict(name="run_db_node_benchmarks", env="SCT_RUN_DB_NODE_BENCHMARKS",
+             type=boolean,
```

Comment:
Did you add description to config.md for new parameter?

Reply:
Good catch :) I'll add the description for that.

Reply:
Description for the param added.